### PR TITLE
[Homework #7] - JS Basics 2 by Ivan Ishchenko

### PR DESCRIPTION
**Pull request by ViVse**

### CODE REVIEW

File: homeworks/ivan-ishchenko/07-js-basics-2/index.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  for (let i = 0; i < triplet.length; i++)
+    if (triplet[i] !== max && triplet[i] !== min) return i;
+}
+
+// task 8 https://www.codewars.com/kata/578553c3a1b8d5c40300037c
+const binaryArrayToNumber = arr => parseInt(arr.join(''), 2);
+
+// task 9 https://www.codewars.com/kata/585d7d5adb20cf33cb000235
+function findUniq(arr) {
+  arr.sort();
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/07-js-basics-2/index.js:60]: Bad practice to mutate an array from params.
Better to create a new one and work inside the function.

File: homeworks/ivan-ishchenko/07-js-basics-2/index.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    // determines what page an item is on. Zero based indexes
+    // this method should return -1 for itemIndex values that are out of range
+    if (itemIndex >= this.itemCount() || itemIndex < 0) return -1;
+    return Math.floor(itemIndex / this.itemsPerPage);
+  }
+}
+
+// task 2 https://www.codewars.com/kata/52597aa56021e91c93000cb0
+const moveZeros = arr => {
+  return arr.filter(e => e !== 0).concat(arr.filter(e => e === 0));
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/07-js-basics-2/index.js:133]: I recommend using the short version of the arrow function without return. It is more readable and reduces the number of code lines.

**Review by VitalikMelnyk — COMMENTED** [review]
Great job, @ViVse!
I have checked your PR, and left some small comments.

**Comment by ViVse** [comment]
@VitalikMelnyk, fixed everything

**Comment by VitalikMelnyk** [comment]
Well Done @ViVse!
Approved.

**Pull request merged**